cluster/get-kube.sh

```diff
@@ -143,10 +144,10 @@ fi
 kernel=$(uname -s)
 case "${kernel}" in
   Darwin)
-    platform="darwin"
+    #platform="darwin"
```

Comment:
Did you mean to comment this (and the next one) out? If so, should we just delete the whole case statement?

Reply:
SGTM, this block of code is copied throughout the codebase currently, but here these values are unused. I'll delete them. (also thanks for review!)

Reply:
done